scripts/verify-allowed-css-variables.mjs

```diff
@@ -0,0 +1,44 @@
+const getDefinedVariables = async () => {
+  const content = await fs.readFile('./packages/theme/theme.css', 'utf-8')
```

Comment:
leave annotation in theme.css that it is used here and should not be moved or renamed

Reply:
also, please add comment for this file - what is it doing?